cogs/games.py

```diff
@@ -5,11 +5,11 @@
 from itertools import islice
 from string import Template
 
-import nextcord
-from nextcord.ext import commands
+import disnake
+from bot_base.paginators.disnake_paginator import DisnakePaginator
```

Comment:
Kinda ugly import rather it be `from bot_base.paginators import DisnakePaginator` but I don't give a shit xD

Reply:
It's a future proofing thing give the bot base needs to work with disnake and nextcord. By importing from the file, and not the module it means if we add more then it won't error because we don't have disnake / nextcord but yee